examples/src/main/scala/example/ziohttp/ExampleApp.scala

```diff
@@ -36,7 +34,7 @@ object ExampleApp extends ZIOAppDefault {
         ZLayer.succeed(
           Server.Config.default
             .port(8088)
-            .withWebSocketConfig(ZHttpAdapter.defaultWebSocketConfig)
+            .webSocketConfig(ZHttpAdapter.defaultWebSocketConfig)
```

Comment:
It's still necessary? I thought it wasn't

Reply:
Unfortunately for now yes, until this makes it into a release: https://github.com/softwaremill/tapir/pull/3291

Reply:
Ah hopefully it comes soon, let's merge this after?

Reply:
Yeah sounds good to me 👍